willibald_dwh/models/rdv_il/lnk_webshoporderitem_delivery.sql

```diff
@@ -0,0 +1,33 @@
+{%- set yaml_metadata -%}
+
+source_model_name: v_webshop_position
+hubs:
+  - name: hub_webshoporderitem
+    bk_columns:
+      - column_name: bestellungid
+        business_name: webshoporder_id
+      - column_name: posid
+        business_name: orderitem_id
+  - name: hub_deliveryaddress
+    bk_columns:
+      - column_name: lieferadrid
+        business_name: deliveryaddress_id
+  - name: hub_deliveryservice
+    bk_columns:
+      - column_name: lieferartid
+        business_name: deliveryservice_id
+transactional_attributes:
+  - column_name: lieferdatum 
```

Comment:
Lieferdatum is not a part of the unique key of webshop_lieferung and might be updated in the source data? I think that is why we modeled it as a link satellite. But if there never is an update we sure can have it as an transactional attribute and do it without the sat.

Reply:
I think you are right with this. Let's stick with the delivery date as a transactional attribute!